lib/onix/contributor.rb

```diff
@@ -6,7 +6,7 @@
 module ONIX
   class Contributor < SubsetDSL
     element "SequenceNumber", :integer, :cardinality => 0..1
-    element "ContributorRole", :subset, :shortcut => :role, :cardinality => 1..n
+    elements "ContributorRole", :subset, :shortcut => :roles, :cardinality => 1..n
```

Comment:
To avoid API break, it would be nice to add a singular `role` method like for `place` https://github.com/immateriel/im_onix/blob/master/lib/onix/contributor.rb#L49